docs/sphinx-docs/source/dev/gl/opengl.rst

```diff
@@ -0,0 +1,12 @@
+Open GL Subsystem
+==============
+
+The SasView openGL subsystem is quite minimal, and works in the standard way though a scenegraph
+
+Within the `visual_checks` directory there are a couple of stand-alone python files that provide
+a way of checking the rendering, and catalogue the available functions
+
+
+Class Heirarchy
+===============
+
```

Comment:
Potentially, depending on where the docs need to go, which I still don't have an answer for. @smk78 ?

Reply:
Assuming the orientation viewer is called from the Tools menu item, it will need an entry in
```
/sasview/docs/sphinx-docs/source/user/tools.rst
```
linking to, I would suggest, _orientation_viewer_help.rst_. By convention, such a file should sit in a _/media_ folder somewhere within one of the _/sasview/src/sas/qtgui_ subfolders. The other Tools are under _/Calculators_ but skimming the Files Changed under this PR it seems most of this work is in _/Utilities_?

But, this is for User Documentation. I don't actually see any user docs in this PR, is that correct? There will need to be some!

Your file _opengl.rst_ is Developer Documentation. That will require an entry in
```
/sasview/docs/sphinx-docs/source/dev/dev.rst
```

BTW, there's a typo on _opengl.rst:_ should be 'Hierarchy', not 'Heirarchy'.

Does this reimplementation of the orientation viewer use any of the sasmodels.jitter functions? If so the dev docs could/should cross-link to the appropriate point in the Sasmodels API documentation, _sasmodels.jitter_.

Reply:
I think full docs needs to be a new PR.

Reply:
Will leave the developers docs for now too.